convention.md

```diff
@@ -125,8 +125,11 @@ If this root topic does not suit your needs (in case of, e.g., a public broker o
 you can change the first segment, but the `"/5/"` segment must be retained. This allows controllers
 to subscribe to only the devices they are compatible with.
 
+## Auto-Discovery
+
 Homie 5 controllers must by default perform auto-discovery on the wildcard topic `"+/5/+/$state"`.
 Controllers are free to restrict discovery to a specific root topic, configurable by the user.
+A zero length payload published on the $state topic indicates a device removal.
```

Comment:
```suggestion
A zero length payload published on the $state topic indicates a device removal. This indicates the device has no intent to return later, and as such controllers should consider all device related (cached) data to be invalid.
```

Reply:
I've given this some more thought, and I think it is actually a matter of a higher level issue; define when a device exists.

I'll drop this, and create an addition to this PR.